cypress/integration/learn/challenges/projects.js

```diff
@@ -49,4 +54,60 @@ describe('project submission', () => {
       // cy.url().should('not.have.string', url);
     });
   });
+  it(
+    'JavaScript projects can be submitted and then viewed in /settings',
+    { browser: 'electron' },
+    () => {
+      cy.login();
+      cy.fixture('../../config/curriculum.json').then(curriculum => {
+        const { challenges, meta } =
+          curriculum['javascript-algorithms-and-data-structures'].blocks[
+            'javascript-algorithms-and-data-structures-projects'
+          ];
+
+        const projectTitles = meta.challengeOrder.map(([, title]) => title);
+        const projectsInOrder = projectTitles.map(projectTitle => {
```

Comment:
Any particular reason for the order?

Reply:
Yep. So that it can check that it's reached the next page before running the next set of tests.